lib/disk/modules/QcowDisk.rb

```diff
@@ -571,6 +620,30 @@ def cluster_offset_mask
     @cluster_offset_mask ||= (1 << csize_shift) - 1
   end
 
+  def incompatible_features
+    version < 3 ? 0 : @header['incompatible_features']
```

Comment:
Would it make sense for this to be more dynamic instead of checking the version?  Something like `@header['incompatible_features'] || 0`